app/models/student.rb

```diff
@@ -17,6 +17,8 @@ class Student < User
   ]
 
   validates :telephone, :city, :province, :university, :language, presence: true
+  validates_format_of :email, with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i
```

Comment:
Prefer the new style validations `validates :column, format: value` over `validates_format_of`.